### ISSUE TITLE
youtube url plugin

### ISSUE THREAD

**Issue by alexey2baranov** [jonschlinkert/remarkable#274]
Hello! 
I found few issues about youtube. Can you say me what is the best plugin to add youtube embed block ?

for example when user write

> watch my video https://www.youtube.com/watch=Lrzr1LlZfhg

i would like to have something like this
```html
<p>watch my video</p>
<you tube html embed/> 
```

**Comment by caraya** [comment]
how about wrapping the URL In a link? 

```markdown
watch my video [https://www.youtube.com/watch=Lrzr1LlZfhg](https://www.youtube.com/watch=Lrzr1LlZfhg)
```

You can also copy the embed from youtube directly. Markdown parsers will honor the HTML and ignore it when converting the rest of the content to html. This has worked for me with one or more embeds on the page

```html
<iframe width="560" height="315" src="https://www.youtube.com/embed/GiCcK039x_o" frameborder="0" allowfullscreen></iframe>
```
Does this help?

**Comment by alexey2baranov** [comment]
@caraya thak you for your answer.

Unfortunally most of our users do not understand markdown rules. What they do is just copy url from address bar and paste it into message field.

I think youtube plugin which i looking for is internal aulolink feature + one extra cheking `if (url.startsWith("youtube"))`. Is anybody writes same pluggin?

**Comment by lpellegr** [comment]
@alexey2baranov Yes, I have created a plugin for this purpose based on a compilation of existing ones:

https://github.com/noticeableapp/remarkable-iframify
